src/main/java/uk/gov/hmcts/reform/pip/data/management/service/publication/PublicationReportingService.java

```diff
@@ -44,6 +49,11 @@ public List<PublicationMiData> getMiData() {
             .atStartOfDay();
         List<PublicationMiData> publicationMiData =
             artefactRepository.getMiData(publicationReceivedDate);
+        List<PublicationMiData> archivedPublicationMiData =
+            artefactArchivedRepository.getArchivedArtefacts(publicationReceivedDate);
+        if (archivedPublicationMiData != null && !archivedPublicationMiData.isEmpty()) {
```

Comment:
Don't think archivedPublicationMiData is ever going to be null? It come back from the query so in theory there will either ever going to be empty array or something in the array.